src/lib/db/project-store.ts

```diff
@@ -178,7 +178,7 @@ class ProjectStore implements IProjectStore {
             .returning(COLUMNS)
             .onConflict('id')
             .ignore();
-        if (rows.length > 0) {
+        if (environments && rows.length > 0) {
```

Comment:
This doesn't affect any test if removed, but at some point, I got an error here and it seems to be fine, if environments is not defined but the SQL statement inserts some rows, the next line will fail, so I think it still makes sense but is one of those unneeded changes that may add noise to our not so small set of changes